Usage docs (#231)
seqeralabs/tower-cli: USAGE.md

Comprehensive details on supported compute environments can be obtained from [Tower Usage docs](https://help.tower.nf/22.1/compute-envs/overview/#introduction).

#### Adding compute-env to a workspace

Once the credentials have been added to a workspace, a Compute Environment (e.g AWS Batch) can be created using those credentials with automatic provisioning of cloud computing resources via **Tower Forge**:

```bash
$ tw compute-envs add aws-batch forge --name=my_aws_ce --credentials=<my_aws_creds_1> --region=eu-west-1 --max-cpus=256 --work-dir=s3://<bucket name> --wait=AVAILABLE

  New AWS-BATCH compute environment 'my_aws_ce' added at user workspace
```

**NOTE**: See the [IAM policy](https://github.com/seqeralabs/nf-tower-aws/tree/master/forge) for Tower Forge for recommendations on AWS Batch permissions, without which the env creation would fail.

The above command will 
- Use the **Tower Forge** mechanism to automatically manage the AWS Batch resource lifesycle (`forge`)
- Use the credentials previously added to the workspace (`--credentials`)
- Create all of the required AWS Batch resources in the AWS Ireland (`eu-west-1`) region 

> - A maximum of 256 CPUs will be provisioned in the compute environment as per the needs of the pipeline (`--max-cpus`)

Reviewer: ```suggestion
- Provision a maximum of 256 CPUs in the Compute Environment (`--max-cpus`)
```

> - An existing S3 bucket will be used as the work directory when running Nextflow (`--work-dir`)

Reviewer: ```suggestion
- Use an existing S3 bucket to store the work directory when running Nextflow (`--work-dir`)
```

> - Will wait until all the resources are AVAILABLE and ready to use (`--wait`)

Reviewer: ```suggestion
- Wait until the Compute Environment has been successfully created and is ready to use (`--wait`)
```

Comprehensive details about Tower Forge are available in the [user documentation](https://help.tower.nf/compute-envs/aws-batch/#forge).

> #### Deleting compute-env to a workspace

Reviewer: ```suggestion
#### Deleting a Compute Environment from a Workspace
```
